chore: add borrower-dashboard category

mcstover opened the pull request:
This has been extensively used in creation of the new borrower-dashboard pages.

mcstover committed:
chore: add borrower-dashboard category

This has been extensively used in creation of the new borrower-dashboard pages.

dyersituations reviewed (approved):
Good call, I missed this one getting through, thanks!

mcstover committed:
chore: add kiva-cards category

This is to support the new Kiva Cards pages and potentially other operations on kiva-cards elsewhere.

eddieferrer commented:
@mcstover I think we can merge this in :)
